PowerScout/AppDelegate.swift

```diff
@@ -13,11 +13,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDele
 
     var window: UIWindow?
     var orientationLock: UIInterfaceOrientationMask = .all
-    var matchStore = MatchStore(withMock: false)
-    var serviceStore = ServiceStore()
+    var matchStore:MatchStore!
+    var serviceStore:ServiceStore!
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
+        // Mock data testing (use this line)
+//        matchStore = MatchStore(withMock: true)
+        // No Mock Data Testing (use this line)
+        matchStore = MatchStore(withMock: false)
```

Comment:
Just like testing the previous transfer PR, You can uncomment the above line to use Mock Data instead of creating it your self. However, we now have the ability to create matches, so testing it with user generated matches is fine too.